extensions/amp-next-page/0.1/next-page-service.js

```diff
@@ -358,13 +355,12 @@ export class NextPageService {
    * the position of a page separator in the viewport.
    * @param {number} i Index of the documentRef this recommendation unit is
    *     attached to.
-   * @param
-   * {!../../../src/service/position-observer/position-observer-worker.PositionInViewportEntryDef}
+   * @param {?../../../src/service/position-observer/position-observer-worker.PositionInViewportEntryDef}
    *     position Position of the current recommendation unit in the viewport.
    */
   positionUpdate_(i, position) {
     // We're only interested when the recommendations exit the viewport
-    if (position.positionRect !== null) {
+    if (!position || position.positionRect !== null) {
```

Comment:
Intended change or out of date base?

Reply:
Intended. See https://github.com/ampproject/amphtml/pull/14439/files#r188090503

Reply:
LGTM